tex/gregoriotex.lua

```diff
@@ -214,7 +214,8 @@ local function init(arg, enable_height_computation)
   if enable_height_computation then
     new_line_heights = {}
 
-    local mcb_version = luatexbase.get_module_version('luatexbase-mcb')
+    local mcb_version = luatexbase.get_module_version and
+        luatexbase.get_module_version('luatexbase-mcb') or 9999
```

Comment:
This is a Lua idiom for a ternary operation.  See http://lua-users.org/wiki/TernaryOperator.  I am using the "standard" solution.  Adding grouping would make it seem more like a boolean operation (which it isn't) so I'm inclined to leave it as-is.